src/client/sandbox/shadow-ui.js

```diff
@@ -28,6 +28,7 @@ export default class ShadowUI extends SandboxBase {
         this.root                    = null;
         this.lastActiveElement       = null;
         this.uiStyleSheetsHtmlBackup = null;
+        this.createWrapperFor        = ShadowUI._createFunctionWrappers();
```

Comment:
Looks good.
`this.wrapperCreators        = ShadowUI._createWrapperCreators();`
@miherlosev What do you think?